src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -186,12 +183,12 @@ export default class FilingHistoryList extends Vue {
    * If there is a filing ID to highlight then it finds it and expands its panel.
    */
   @Watch('getFilings', { immediate: true })
-  private onFilingsChange (): void {
+  onFilingsChange (): void {
     // if needed, highlight a specific filing
     if (this.highlightId) {
       const index = this.getFilings.findIndex(f => (f.filingId === this.highlightId))
       if (index >= 0) { // safety check
-        this.toggleFilingHistoryItem(index, this.getFilings[index])
+        this.toggleFilingHistoryItem(index)
```

Comment:
Eslint complained that this function should only have one argument but it got 2. So, I removed the second argument.

Reply:
Must have been left over from my refactor. Looks good 👍 

Do a global search for `toggleFilingHistoryItem` -- there are 4 calls to this with a second parameter -- please fix those, too. Thx.

Reply:
Fixed, thanks.